src/Samples/Samples.Wpf.MVVM/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,16 @@
+using GalaSoft.MvvmLight; /* Or your MVV provider of choice */
```

Comment:
I'd prefer a sample that builds out of the box, without installing a MVVM provider. Would that be possible to write a INotifyPropertyChanged implementation directly instead?

Reply:
Does it work for you now?